src/pages/SettingsPage.js

```diff
@@ -68,7 +94,7 @@ const SettingsPage = ({
             >
                 <View style={styles.settingsWrapper}>
                     <View
-                        style={[styles.largeAvatar, styles.mb3]}
```

Comment:
this was a typo, it doesn't exist. There is a similar class name to this but it already gets applied within the component